### DISCUSSION TITLE
materials variants extension on chrome ios ar

### DISCUSSION THREAD

**Discussion by codedotart** [google/model-viewer#2553] [Q&A]
Hi, 

I wanted to check if it is possible to use the materials variant feature on chrome ios. It works well on safari but the ar doesn't work on chrome browser. I believe the usdz auto-conversion feature doesn't work on chrome ios? Or am I doing anything wrong here?

Here is the viewer we made:
https://teal-whip-brie.glitch.me/

**Comment by elalish** [comment]
Correct, it's a Chrome iOS bug: #2449
